roles/kubernetes/control-plane/tasks/main.yml

```diff
@@ -94,6 +94,7 @@
     - k8s-certs-renew.timer
   register: k8s_certs_units
   when: auto_renew_certificates
+  # TODO
```

Comment:
To do what?

Reply:
In the case of a external CA managed certs I think auto_renew_certificates should be forcefully set to False, no?

Reply:
Yes I already changed this. I have added a task in roles/kubernetes/preinstall/tasks/0020-verify-settings.yml to check the value of `auto_renew_certificates` when `kube_external_ca_mode` is set to true. It stops the execution when it is inconsistent unless `ignore_assert_errors` is set to true.